applications/MultilevelMonteCarloApplication/python_scripts/montecarlo_analysis.py

```diff
@@ -0,0 +1,224 @@
+        self._GetSolver().main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_AREA)
+
+    def _CreateSolver(self):
+        import convection_diffusion_stationary_solver
+        solver = convection_diffusion_stationary_solver.CreateSolver(self.model,self.project_parameters["solver_settings"])
+        self.LaplacianSolver = solver
+        return self.LaplacianSolver
+    
+    def _GetSimulationName(self):
+        return "Monte Carlo Analysis"
+
+    '''Introduce here the stochasticity in the right hand side defining the forcing function and apply the stochastic contribute'''
+    def ModifyInitialProperties(self):
+        for node in self.model.GetModelPart("MLMCLaplacianModelPart").Nodes:
+            coord_x = node.X
+            coord_y = node.Y
+            # forcing = -432.0 * coord_x * (coord_x - 1) * coord_y * (coord_y - 1)
+            forcing = -432.0 * (coord_x**2 + coord_y**2 - coord_x - coord_y) # this forcing presents an analytical solution
+            node.SetSolutionStepValue(KratosMultiphysics.HEAT_FLUX,forcing*self.sample)
+
+
+##################################################
+######## END OF CLASS MONTECARLOANALYSIS #########
+##################################################
+
+
+'''
+function generating the random sample
+here the sample has a beta distribution with parameters alpha = 2.0 and beta = 6.0
+'''
+def GenerateBetaSample():
+    alpha = 2.0
+    beta = 6.0
+    number_samples = 1
+    sample = np.random.beta(alpha,beta,number_samples)
+    return sample
+
+
+'''
+function evaluating the QoI of the problem: int_{domain} TEMPERATURE(x,y) dx dy
+right now we are using the midpoint rule to evaluate the integral: improve!
+'''
+def EvaluateQuantityOfInterest(simulation):
+    """here we evaluate the QoI of the problem: int_{domain} SOLUTION(x,y) dx dy
+    we use the midpoint rule to evaluate the integral"""
+    KratosMultiphysics.CalculateNodalAreaProcess(simulation._GetSolver().main_model_part,2).Execute()
+    Q = 0.0
+    for node in simulation._GetSolver().main_model_part.Nodes:
+        Q = Q + (node.GetSolutionStepValue(KratosMultiphysics.NODAL_AREA)*node.GetSolutionStepValue(KratosMultiphysics.TEMPERATURE))
+        #print("NODAL AREA = ",node.GetSolutionStepValue(KratosMultiphysics.NODAL_AREA),"NODAL SOLUTION = ",node.GetSolutionStepValue(KratosMultiphysics.TEMPERATURE),"CURRENT Q = ",Q)
+    return Q
+
+
+'''
+function executing the problem
+input:
+        model_part_file_name : path of the model part file (still to implement how to import in efficient way in a loop where I have different model part files and different ProjectParameters files, thus for now read model part name from the ProjectParameters.json file)
+        parameter_file_name  : path of the Project Parameters file
+        sample               : stochastic random variable
+output:
+        QoI                  : Quantity of Interest
+'''
+@task(model_part_file_name=FILE_IN, parameter_file_name=FILE_IN, returns=1)
```

Comment:
OK, but as a todo we need to use the serialization mechanism (not for this PR, for the future)